src/components/BookDisplay.js

```diff
@@ -12,19 +13,23 @@ const BookDisplay = () => {
     dispatch(getBook());
   }, [dispatch]);
   return (
-    <div>
-      {Booklists.map((item) => (
-        <div key={item.item_id} className="bookCar">
-          <Book
-            title={item.title}
-            author={item.author}
-            id={item.item_id}
-            category={item.category}
-          />
-        </div>
-      ))}
-      <InputBooks />
-    </div>
+    <section className="section">
+      <div className="book-part">
+        {Booklists.map((item) => (
+          <div key={item.item_id} className="bookCar">
+            <Book
+              title={item.title}
+              author={item.author}
+              id={item.item_id}
+              category={item.category}
+            />
+          </div>
+        ))}
+      </div>
+      <div className="input">
+        <InputBooks />
+      </div>
+    </section>
```

Comment:
- [ ] Kindly use the correct background color so that your design can match the desired design provided in Zeplin.

**This is how your design looks like:**

![image](https://user-images.githubusercontent.com/86318284/202716384-adab841f-faad-4390-8693-9abe5b065aab.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/202716461-d4c4db90-583e-426e-bf6f-c3db27a6d17d.png)